src/Sarif/Errors.cs

```diff
@@ -46,8 +46,10 @@ public static void LogExceptionLoadingTarget(IAnalysisContext context)
                 throw new ArgumentNullException(nameof(context));
             }
 
+            context.RuntimeErrors |= RuntimeConditions.ExceptionLoadingTargetFile;
```

Comment:
>context [](http://example.com/codeflow?start=12&length=7)

We now set RuntimeErrors early in every method, so that this data is persisted in the event that an unhandled exception occurs in the remainder of the method (as happened during my current dev/test loop).

Reply:
Curious what exception you got, guess its thrown by logger? Should we apply the same change to `Warnings`?